libandroid-navigation-ui/src/main/java/com/mapbox/services/android/navigation/ui/v5/NavigationView.java

```diff
@@ -253,7 +247,13 @@ public boolean isRecenterButtonVisible() {
 
   @Override
   public void drawRoute(DirectionsRoute directionsRoute) {
-    navigationMap.drawRoute(directionsRoute);
+    if (navigationMap != null) {
```

Comment:
@Guardiola31337 let's double check that all `public` `NavigationView` APIs that use the `NavigationMapboxMap` are protected like this one